src/app/components/header/header.component.html

```diff
@@ -1,5 +1,9 @@
 <mat-toolbar color="primary" class="space-between">
-  <div><a id="headerTitleText" [routerLink]="ROUTES.HOME | route">MAPS</a></div>
+  <div class="header-logo">
+    <a [routerLink]="ROUTES.HOME | route">
+      <img src="../../../assets/images/maps-logo-minimal-white.svg" alt="BMGF MAPS Logo" class="logo">
```

Comment:
Would this be better as '/assets/images...'?

Reply:
I didn't really get from issue #293 what the "full" logo was supposed to be for.  Do you?

Reply:
Is the "header-logo" div needed?

Reply:
1. Indeed not sure why I didn't add it that way, might have left it when I was trying different things because it wasn't working
2. I think it comes down to the logo being too detailed for how small it is in the website so the idea was to use a more simplified "non full logo" and also the colors made it hard to read. So Paulius made different variants based off those criteria
3. Nope removed it :)